GP03FLMA/GP03FLMA_D00005_ingest.csv

```diff
@@ -1,20 +1,20 @@
+#,/omc_data/whoi/OMC/GP03FLMA/D00005/*ctdmo_*.dat,GP03FLMA-RIM01-02-CTDMOG040,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
+#,/omc_data/whoi/OMC/GP03FLMA/D00005/*ctdmo_*.dat,GP03FLMA-RIM01-02-CTDMOG041,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
+#,/omc_data/whoi/OMC/GP03FLMA/D00005/*ctdmo_*.dat,GP03FLMA-RIM01-02-CTDMOG042,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
+#,/omc_data/whoi/OMC/GP03FLMA/D00005/*ctdmo_*.dat,GP03FLMA-RIM01-02-CTDMOG043,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
+#,/omc_data/whoi/OMC/GP03FLMA/D00005/*ctdmo_*.dat,GP03FLMA-RIM01-02-CTDMOG044,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
+#,/omc_data/whoi/OMC/GP03FLMA/D00005/*ctdmo_*.dat,GP03FLMA-RIM01-02-CTDMOG045,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
+#,/omc_data/whoi/OMC/GP03FLMA/D00005/*ctdmo_*.dat,GP03FLMA-RIM01-02-CTDMOG046,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
+#,/omc_data/whoi/OMC/GP03FLMA/D00005/*ctdmo_*.dat,GP03FLMA-RIM01-02-CTDMOG047,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
+#,/omc_data/whoi/OMC/GP03FLMA/D00005/*ctdmo_*.dat,GP03FLMA-RIM01-02-CTDMOG048,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
+#,/omc_data/whoi/OMC/GP03FLMA/D00005/*ctdmo_*.dat,GP03FLMA-RIM01-02-CTDMOH049,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
+#,/omc_data/whoi/OMC/GP03FLMA/D00005/*ctdmo_*.dat,GP03FLMA-RIM01-02-CTDMOH050,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
+#,/omc_data/whoi/OMC/GP03FLMA/D00005/*ctdmo_*.dat,GP03FLMA-RIM01-02-CTDMOH051,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.sio_eng.sio.sio_eng_sio_telemetered_driver,/omc_data/whoi/OMC/GP03FLMA/D00005/*status_1487601.dat,GP03FLMA-RIS01-00-SIOENG000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)

```

Comment:
PAPA FLM-A secondary controller 14874 for D5

Reply:
Agreed. Will update.